src/Docs/Resources/current/2-internals/1-core/20-data-abstraction-layer/020-search.md

```diff
@@ -37,6 +37,14 @@ $criteria->addFilter(
 );
 ```
 
+### Adding associations to your criteria
+
+By default your query will only fetch the main entity you are requesting. You can add associations to your criteria object in order to fetch data for that association.
+
+```php
+$orderCriteria->addAssociation('lineItems');
```

Comment:
Could you also add an example and a little explanatory text on how to add nested associations?

Reply:
No problem at all. I will update the PR with an example. Can you explain by what you mean by a nested association? 

Do you mean when say you query the order repository that you need to associations for transactions?

Reply:
Sorry I think I know what you mean now about nested associations. So for instance if I was to add a transaction association to the the query for an order it won't load the transactions associations unless I add for instance `$criteria->addAssociationPath('transactions.paymentMethod');` ?

Reply:
I mean how you would add multiple levels of associations, e.g. if you search the orders and need the deliveries, as well as the positions of the deliveries and then the lineItem of the position would  translate to
```php
$orderCriteria->addAssociation('deliveries.positions.oderLineItem');
```

Reply:
> `$criteria->addAssociationPath('transactions.paymentMethod');` ?

the `addAssociationPath()` method doesn't exist anymore, `addAssociation()` can now handle both cases

Reply:
https://github.com/shopware/platform/blob/master/UPGRADE-6.1.md#core
It's the second point in the `UPGRADE.md`

Reply:
I have updated the documentation to include an example for a simple and nested association. Please let me know if you need any further changes :)